Flaky Zebra test

There is currently a test that's flaky:
```
FAILED src/artemis/device_setup_plans/unit_tests/test_zebra_setup.py::test_zebra_cleanup - AssertionError: assert 0 == 36
 +  where 0 = <bound method FakeEpicsSignal.get of FakeEpicsSignal(name='zebra_output_out_2', parent='zebra_output', value=36, timestamp=1690891786.051812)>()
 +    where <bound method FakeEpicsSignal.get of FakeEpicsSignal(name='zebra_output_out_2', parent='zebra_output', value=36, timestamp=1690891786.051812)> = FakeEpicsSignal(name='zebra_output_out_2', parent='zebra_output', value=36, timestamp=1690891786.051812).get
==== 1 failed, 180 passed, 2 skipped, 31 deselected, 139 warnings in 22.47s ====
```
 Fix it

--- a/src/artemis/device_setup_plans/unit_tests/test_zebra_setup.py
+++ b/src/artemis/device_setup_plans/unit_tests/test_zebra_setup.py
@@ -1,5 +1,5 @@
 from functools import partial
-from unittest.mock import MagicMock, patch
+from unittest.mock import MagicMock
 
 import pytest
 from bluesky.run_engine import RunEngine
@@ -35,15 +35,13 @@ def zebra():
     return i03.zebra(fake_with_ophyd_sim=True)
 
 
-@patch("bluesky.plan_stubs.wait", autospec=True)
-def test_zebra_set_up_for_fgs(bps_wait, RE, zebra: Zebra):
+def test_zebra_set_up_for_fgs(RE, zebra: Zebra):
     RE(setup_zebra_for_fgs(zebra, wait=True))
     assert zebra.output.out_pvs[TTL_DETECTOR].get() == IN3_TTL
     assert zebra.output.out_pvs[TTL_SHUTTER].get() == IN4_TTL
 
 
-@patch("bluesky.plan_stubs.wait", autospec=True)
-def test_zebra_set_up_for_rotation(bps_wait, RE, zebra: Zebra):
+def test_zebra_set_up_for_rotation(RE, zebra: Zebra):
     RE(setup_zebra_for_rotation(zebra, wait=True))
     assert zebra.pc.gate_trigger.get(as_string=True) == I03Axes.OMEGA.value
     assert zebra.pc.gate_width.get() == pytest.approx(360, 0.01)
@@ -51,8 +49,7 @@ def test_zebra_set_up_for_rotation(bps_wait, RE, zebra: Zebra):
         RE(setup_zebra_for_rotation(zebra, direction=25))
 
 
-@patch("bluesky.plan_stubs.wait", autospec=True)
-def test_zebra_cleanup(bps_wait, RE, zebra: Zebra):
+def test_zebra_cleanup(RE, zebra: Zebra):
     RE(set_zebra_shutter_to_manual(zebra, wait=True))
     assert zebra.output.out_pvs[TTL_DETECTOR].get() == PC_PULSE
     assert zebra.output.out_pvs[TTL_SHUTTER].get() == OR1

--- a/src/artemis/system_tests/test_device_setups_and_cleanups.py
+++ b/src/artemis/system_tests/test_device_setups_and_cleanups.py
@@ -32,20 +32,20 @@ def connected_zebra():
 
 @pytest.mark.s03
 def test_zebra_set_up_for_fgs(RE, connected_zebra: Zebra):
-    RE(setup_zebra_for_fgs(connected_zebra))
+    RE(setup_zebra_for_fgs(connected_zebra, wait=True))
     assert connected_zebra.output.out_pvs[TTL_DETECTOR].get() == IN3_TTL
     assert connected_zebra.output.out_pvs[TTL_SHUTTER].get() == IN4_TTL
 
 
 @pytest.mark.s03
 def test_zebra_set_up_for_rotation(RE, connected_zebra: Zebra):
-    RE(setup_zebra_for_rotation(connected_zebra))
+    RE(setup_zebra_for_rotation(connected_zebra, wait=True))
     assert connected_zebra.pc.gate_trigger.get(as_string=True) == I03Axes.OMEGA.value
     assert connected_zebra.pc.gate_width.get() == pytest.approx(360, 0.01)
 
 
 @pytest.mark.s03
 def test_zebra_cleanup(RE, connected_zebra: Zebra):
-    RE(set_zebra_shutter_to_manual(connected_zebra))
+    RE(set_zebra_shutter_to_manual(connected_zebra, wait=True))
     assert connected_zebra.output.out_pvs[TTL_DETECTOR].get() == PC_PULSE
     assert connected_zebra.output.out_pvs[TTL_SHUTTER].get() == OR1

